schematic/manifest/generator.py

```diff
@@ -782,13 +782,16 @@ def set_dataframe_by_url(
         # column-set of the existing manifest so that the user can modify their data if needed 
         # to comply with the latest schema
 
-        # get headers from existing manifest and sheet
+        # get headers from existing manifest and sheet 
```

Comment:
If you run the black formatter on the repo, these errant spaces should be stripped.